catalog/output/outbreaks.json

```diff
@@ -731,6 +731,11 @@
     "name": "Paramyxoviridae",
     "priority": "HIGH",
     "resources": [
+      {
+        "title": "hgPhyloPlace | Place you sequences in a global phylogenetic tree (Measles)",
+        "url": "https://genome.ucsc.edu/cgi-bin/hgPhyloPlace?hgpp_org=hub_6572355_GCF_000854845.1",
```

Comment:
That seems to drop me into a page that has sars-cov-2 pre-selected, and the database below that also only mentions sars-cov-2. Is that how it should be ?
<img width="1039" height="679" alt="Screenshot 2025-07-23 at 17 22 58" src="https://github.com/user-attachments/assets/38f6236f-a544-4a19-8023-f8492d427a21" />